src/fedora/39/amd64/Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM library/fedora:39
+FROM fedora:39
```

Comment:
Using `library` is necessary. See https://github.com/dotnet/docker-tools/issues/1341.

Reply:
That was my fault.  I had tested locally and removed to workaround https://github.com/dotnet/docker-tools/issues/1363. 
 I forgot to add it back once done.

Reply:
Fixed in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1226/commits/2ffb559d061bda985f9a6a67b980b8f71a992852